sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -41,6 +43,13 @@ public boolean isEnabled() {
   SentryClient(final @NotNull SentryOptions options) {
     this.options = Objects.requireNonNull(options, "SentryOptions is required.");
     this.enabled = true;
+    // TODO: how to decide if session aggregates should be used?
+    if (options.isAutoSessionTracking() && options.getRelease() != null) {
```

Comment:
What's the right way to decide if server or client side sessions should be used? One way is to have an option on `SentryOptions` (I don't think though that `auto-session-tracking` is the right one), anther way is to have it on the `Session` object level. Then `Hub#startSession` method would get a parameter `mode` with possible values `application`, `request`.

Reply:
Not valid anymore.